templates/solutions/ai-infrastructure/index.html

```diff
@@ -252,16 +252,16 @@ <h2>Accelerate innovation with certified hardware</h2>
           Get peace of mind with Ubuntu certified hardware from all major OEMs. Choosing validated platforms significantly reduces deployment time and costs for end-customers, in addition to improving reliability and providing bug-fixes faster.
         </p>
         <p class="p-section--shallow">
-          Combined with NVIDIA DGX systems or NVIDIA-Certified systems, choosing an Ubuntu-certified platform provides secure, performant hardware that is guaranteed to deploy quickly and run smoothly.
+          For example, combined with NVIDIA DGX systems or NVIDIA-Certified systems, choosing an Ubuntu-certified platform provides secure, performant hardware that is guaranteed to deploy quickly and run smoothly.
         </p>
         <hr class="is-muted" />
         <p>
-          <a href="/partners/silicon">Explore our silicon programme&nbsp;&rsaquo;</a>
+          <a href="https://ubuntu.com/certified">Explore our certified hardware&nbsp;&rsaquo;</a>
```

Comment:
```suggestion
          <a href="https://ubuntu.com/certified">Explore certified hardware&nbsp;&rsaquo;</a>
```
Copy doc seems to call for "Explore certified hardware" rather than "Explore our certified hardware", just remove the "our"
![image](https://github.com/canonical/canonical.com/assets/46915153/eed6fa85-28fe-488c-8802-ddc1872e4aa9)